lib/cache-save.js

```diff
@@ -10,6 +10,8 @@ async function saveCache() {
 
     if (primaryKey !== matchedKey) {
       await cache.saveCache([await getCacheDirectory()], primaryKey);
+    } else if (primaryKey === "" && matchedKey === "") {
+      core.info(`No cache to validate from.`);
```

Comment:
What does it mean and why we add this now? 🤔

Reply:
If somehow there was no `Scarb.lock` file found, then `primaryKey` and `matchedKey` would be empty strings and the message would say that the cache hit occured - IMO it is a bit misleading, though it doesn't change anything